### ISSUE TITLE
I have tried to install LitMod2D_2.0 on Ubuntu 20.04. While run the command “pip install -r requirements.txt”, I got an error at pyside, traits, and traitsui packages installation. Please find the attached screenshots below

### ISSUE THREAD

**Issue by ramaarksuri** [ajay6763/LitMod2D_2.0_package_dist_users#16]
![Screenshot from_1](https://user-images.githubusercontent.com/41540558/118252528-7be40f00-b4c6-11eb-8d2d-8038ef80abce.png)

# I have also tried "sudo easy_install pyside". At this time I got error like setup script exited with error: failed to find cmake.

![Screenshot from_2](https://user-images.githubusercontent.com/41540558/118252987-0593dc80-b4c7-11eb-831b-14501e6768d1.png)

Please help me to install LitMod2D_2.0 on Ubuntu 20.04

**Comment by wzhang1994** [comment]
Yes, it is not easy to install PySide and LitMod2D_2.0 using pip. Here, you can find a possible solution at https://github.com/ajay6763/LitMod2D_2.0_package_dist_users/issues/8.

If not working, I recommend using the docker way in GUI Manual.
